lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "packages": ["packages/*", "examples/*"],
+  "packages": ["packages/*", "examples/*", "examples/persisted-queries/*"],
```

Comment:
why  this change? doesn't examples/* capture this already?

Reply:
the `persisted-queries` folder has a `client` and `server` package subfolders, so they are not covered with the existing glob, was hitting https://github.com/lerna/lerna/issues/2127 with other setups